### DISCUSSION TITLE
Call for Participation: 2023 Summit

### DISCUSSION THREAD

**Discussion by emilymbender** [delph-in/docs#113] [Announcements]
Dear all,

The 2023 Summit will be hosted by UDC in A Coruña, Spain, from June 26-30. You can find information about the event at this wiki page:

https://github.com/delph-in/docs/wiki/GaliciaTop

We are looking forward to seeing many of you there in person + many virtually as well. And a huge thank you to Olga for hosting us! 

The program chairs will be requesting program input in due course. For now, please register your intent to attend (and whether you plan to be in person or virtual) on the GaliciaParticipants page:

https://github.com/delph-in/docs/wiki/GaliciaParticipants

Finally, please consider who else at your site might be interested in participating and let them know too.

Emily,
for the Standing Committee

**Comment by olzama** [comment]
Dear all @delph-in/participants, there appears to be a slight disaster in the sense that I first indicated "July" at https://github.com/delph-in/docs/wiki/GaliciaParticipants when I put my name there. The correct days are of course JUNE, as indicated on https://github.com/delph-in/docs/wiki/GaliciaTop 

I sincerely hope people who have already registered haven't made accommodations/bought tickets for July?!

**Reply by olzama** [reply]
I have corrected all mentions of "July" to "June" to avoid any further confusion, and have contacted all people who have already registered. Please let me know if you are having any issue due to the confusion that happened.

**Reply by goodmami** [reply]
> I have corrected all mentions of "June" to "July"

I think you mean 'all mentions of "July" to "June"'?

**Reply by olzama** [reply]
Aaaaaargh! Yes. Thank you Michael!
